extractor/src/main/java/com/example/extractor/MainLauncher.java

```diff
@@ -0,0 +1,78 @@
+package com.example.extractor;
+
+import java.nio.file.*;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import java.util.regex.Pattern;
+import java.util.regex.Matcher;
+
+public class MainLauncher {
+
+    public static void main(String[] args) throws Exception {
+        // Input and output directories
+        String inputDirectory = ".ci-temp/checkstyle/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/abbreviationaswordinname";
+        String outputDirectory = "generated/AbbreviationAsWordInName";
```

Comment:
`String outputDirectory = "AbbreviationAsWordInName";`

you should override content of existing folder.